notebooks/working/_03-tb-deconfounder-predictive-checks.py

```diff
@@ -534,48 +301,23 @@ def perform_visual_predictive_cit_test(
     (x_standardized_samples_post *
      drive_alone_stds[None, :, None] +
      drive_alone_means[None, :, None])
-# -
 
 # Look at the dimensions of the prior predictive samples
 print(x_samples_post.shape)
+# -
 
 # ### Visualize the posterior predictive distribution
 
 # +
-# Visualize the prior predictive ditributions
-from viz.sim_cdf import _plot_single_cdf_on_axis
-
-# Choose colors
-orig_color = '#045a8d'
-sim_color = '#a6bddb'
-
 # Choose a column of data to compare
 current_col = 0
 
-# Create the figure
-fig, ax = plt.subplots(figsize=(10, 6))
-
-# Plot the simulated cdfs
-for sim_id in range(NUM_PRIOR_SAMPLES):
-    label = 'Simulated' if sim_id == 0 else None
-    _plot_single_cdf_on_axis(
-        x_samples_post[:, current_col, sim_id], ax, color=sim_color, label=label, alpha=0.5)
-    
-# Plot the observed cdf
-_plot_single_cdf_on_axis(
-    drive_alone_df.iloc[:, current_col],
-    ax, color=orig_color, label='Observed', alpha=1.0)
-
-# Label the plot axes
-ax.set_xlabel("{}".format(drive_alone_variables[current_col]), fontsize=12)
-ax.set_ylabel("Cumulative\nDensity\nFunction",
-              rotation=0, labelpad=40, fontsize=12)
-
-# Show the line labels
-plt.legend(loc='best')
-
-sbn.despine()
-fig.show()
+posterior_sim_cdf =\
+    li.plot_simulated_vs_observed_cdf(
+        drive_alone_df.iloc[:, current_col].values,
+        x_samples_post[:, current_col, :],
+        x_label=drive_alone_variables[current_col]
+        )
 
 # +
 # Compare a single sample from the prior and posterior distributions
```

Comment:
Any benefit to creating the dictionary as a separate value then feeding it to pd.DataFrame?

Reply:
I don't think so because I never reuse the dictionary values. What value do you see in doing so? It's possible I'm overlooking something!